src/host/libpixyusb/src/pixyinterpreter.cpp

```diff
@@ -162,7 +175,7 @@ void PixyInterpreter::interpreter_thread()
 
     // Mutual exclusion for receiver_ object (Unlock) //
     chirp_access_mutex_.unlock();
-    usleep(15000); // wait for 15ms, ie give time for 
+    std::this_thread::sleep_for(std::chrono::milliseconds(15)); // wait for 15ms, ie give time for
   }
```

Comment:
This is pre-existing: the comment is incomplete. Could you add words or remove the second half of it?

Reply:
done